tekton-lint/Dockerfile

```diff
@@ -1,11 +1,11 @@
-FROM registry.fedoraproject.org/fedora:35
+FROM registry.fedoraproject.org/fedora:40
```

Comment:
FYI 41 was just released, 40 is fine too though

Reply:
added F41